plugin-build/src/main/kotlin/io/sentry/android/gradle/telemetry/SentryTelemetryService.kt

```diff
@@ -495,6 +507,9 @@ abstract class SentryCliInfoValueSource : ValueSource<String, InfoParams> {
 
 abstract class SentryCliVersionValueSource : ValueSource<String, VersionParams> {
     interface VersionParams : ValueSourceParameters {
+        @get:Input
+        val projectBuildFolder: Property<File>
```

Comment:
```suggestion
        val buildDirectory: Property<File>
```
should we align naming with the one above?